Domaciukoly20/driving_cars.py

```diff
@@ -0,0 +1,54 @@
+import time
+
+class Car:
+    def __init__(self, economy, color, model, mileage=0, fuel=100, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
super call is extra. it is needed only when you have a parent class and doing inherence of it.
**kwargs - is also extra. we want to strict user from inputting un-expected values here